.circleci/config.yml

```diff
@@ -1478,45 +1496,97 @@ workflows:
     when:
       or:
         - matches:
-            pattern: "^release/.*$"
+            pattern: "^(release/.*|main)$"
             value: << pipeline.git.branch >>
-        - equal:
-            - "run-manual-tests"
-            - << pipeline.parameters.action >>
-        - equal:
-            - "run-from-github-comments"
-            - << pipeline.parameters.GHA_Meta >>
+        - equal: ["run-manual-tests", << pipeline.parameters.action >>]
+        - equal: ["run-from-github-comments", << pipeline.parameters.GHA_Meta >>]
     jobs:
-      - backend-integration-tests-SK1
-      - backend-integration-tests-SK2
-      - backend-integration-tests-custom-entitlements
-      - backend-integration-tests-other
-      - build-tv-watch-and-macos
-      - build-visionos
-      - docs-build
-      - run-test-ios-14
-      - run-test-ios-15
-      - run-test-ios-16
-      - run-test-macos
-      - run-test-tvos
-      - run-test-watchos
-      - spm-receipt-parser
-      - spm-release-build
-      - spm-release-build-xcode-14
-      - spm-release-build-xcode-15
-      - spm-revenuecat-ui-ios-15
-      - spm-revenuecat-ui-ios-16
-      - run-revenuecat-ui-ios-17
-      - run-revenuecat-ui-ios-18
-      - spm-revenuecat-ui-watchos
-      - installation-tests-cocoapods
-      - installation-tests-swift-package-manager
-      - installation-tests-custom-entitlement-computation-swift-package-manager
-      - installation-tests-carthage
-      - installation-tests-xcode-direct-integration
-      - installation-tests-receipt-parser
-      - api-tests
+      - backend-integration-tests-SK1:
```

Comment:
a bit hard to review but:
- I added the context to all
- Removed `run-revenuecat-ui-ios-17` and `run-revenuecat-ui-ios-18` and `emerge_purchases_ui_snapshot_tests` which are already run in the `build-test` workflow and should speed things up
- Added `backend-integration-tests-offline` which was missing